engine/js/BertaGalleryRow.js

```diff
@@ -169,101 +170,88 @@ var BertaGalleryRow = new Class({
     liElement.addClass('selected');
   },
 
-  load: function (src, mType, mWidth, mHeight, videoPath, autoPlay, caption, bDeleteExisting, xImgIndex, srcset) {
+  load: function (src, mType, mWidth, mHeight, videoPath, autoPlay, caption, xImgIndex, srcset) {
+    this.currentItem += 1;
+    this.itemIsLoaded = this.currentItem <= this.loadedItems;
```

Comment:
let's call this `currentItemIsLoaded`. Otherwise, it's not clear, what is the loaded item since the gallery class handles all of the items.